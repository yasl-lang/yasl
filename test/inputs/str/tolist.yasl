echo 'abcdefg'->tolist()
echo 'abcdefg'->tolist(2)
