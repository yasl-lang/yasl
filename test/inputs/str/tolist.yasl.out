[a, b, c, d, e, f, g]
[ab, cd, ef, g]
